dwave/plugins/sklearn/utilities.py

```diff
@@ -47,10 +47,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from itertools import combinations
 import typing
 
+from joblib import Parallel, delayed, cpu_count
```

Comment:
Is `joblib` a dependency of `scikit-learn`? Even if that is so, if we use it directly we need to add it to `setup.cfg` and `requirements.txt`.

Reply:
importing joblib from sklearn now